libzim/wrapper.pyx

```diff
@@ -414,14 +417,19 @@ cdef class FilePy:
             The ZimFileArticle object
         Raises
         ------
-            RuntimeError
+            NotFound
                 If an article with the provided id is not found in the file
+                Or if the provided id is out of bound
         """
 
         # Read to a zim::Article
-        cdef wrapper.Article art = self.c_file.getArticle(<int> id)
+        cdef wrapper.Article art
+        try:
+            art = self.c_file.getArticle(<int> id)
+        except RuntimeError as exc:
+            raise NotFound(str(exc))
```

Comment:
If you have an exception here, it is about something else than a non existing article.
I would raise a `RuntimeError`.

Reply:
It's usually because of an out-of-bound id. We can raise `RuntimeError` for that.